### ISSUE TITLE
glob pattern does not work in "ignoreWorkspaces" option

### ISSUE THREAD

**Issue by eltonio450** [webpro/knip#174]
Hello

Using glob expression does not work in ignoreWorkspaces option, for example, `ignoreWorkspaces: ["packages/*"]`

Example of a basic monorepo: https://github.com/eltonio450/knip-glob

In this monorepo:

```
const config = {
    ignoreWorkspaces: ["packages/*"]
}

export default config
```
outputs: 
<img width="322" alt="image" src="https://github.com/webpro/knip/assets/6675365/e40fc23d-8e84-4099-baaa-a2cfb98d590f">

whereas: 
```
const config = {
    ignoreWorkspaces: ["packages/1"]
}

export default config
```
outputs:
<img width="288" alt="image" src="https://github.com/webpro/knip/assets/6675365/1cec33d2-d77d-45e7-a737-1fd296e9eaf8">

Not sure if it's a feature or a bug, but as we had to add this option when migrating to v2, it's kinda to add all the packages manually in this option

Thank you !

**Comment by webpro** [comment]
Now I see unfortunately the documentation indicates wildcards are allowed in `ignoreWorkspaces`, while they're not. So it's a feature request after all.

**Comment by eltonio450** [comment]
"it's not", meaning it's not meant to be for now ?

thank you !

**Comment by webpro** [comment]
Yeah, it's not meant to be today :)

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v2.17.2. See [Release 2.17.2](https://github.com/webpro/knip/releases/tag/2.17.2) for release notes._

**Comment by eltonio450** [comment]
whoa, that was fast, thanks !
